docs/website/docs/reference/bindings/python.md

```diff
@@ -188,3 +188,12 @@ IREE_PY_RUNTIME=tracy iree-run-module ...
 See the developer documentation page on
 [Profiling with Tracy](../../developers/performance/profiling-with-tracy.md)
 for information on using Tracy.
+
+!!! tip - "Tip - flushing profile data"
+    When writing a Python-based program that you want to profile you may need to
```

Comment:
I _think_ this might need a newline. If you want to preview, https://github.com/openxla/iree/tree/main/docs/website#how-to-edit-this-documentation has instructions.

```suggestion
!!! tip - "Tip - flushing profile data"

    When writing a Python-based program that you want to profile you may need to
```